x-pack/solutions/search/plugins/search_assistant/public/components/nav_control/index.tsx

```diff
@@ -147,7 +140,11 @@ export function NavControl() {
   );
 }
 
+const isMac = navigator.userAgent.toLocaleLowerCase().includes('mac');
 const buttonLabel = i18n.translate(
   'xpack.searchAssistant.navControl.openTheAIAssistantPopoverLabel',
-  { defaultMessage: 'Open the AI Assistant' }
+  {
+    defaultMessage: 'Keyboard shortcut {keyboardShortcut}',
+    values: { keyboardShortcut: isMac ? '⌘ ;' : 'Ctrl ;' },
```

Comment:
_for reference_ similar changes in security : https://github.com/elastic/kibana/blob/main/x-pack/platform/packages/shared/kbn-elastic-assistant/impl/assistant_context/assistant_nav_link.tsx#L16